config/spryk/templates/Zed/Business/MethodBusinessFactory.php.twig

```diff
@@ -15,13 +15,13 @@
     {
         return new \{{ organization }}\Zed\{{ module }}\Business\{% if subDirectory %}{{ subDirectory | replace({"/": "\\"}) }}\{% endif %}{{ className }}(
         {% for dependencyMethod in dependencyMethods.value -%}
-            {%- spaceless -%}
+            {%- apply spaceless -%}
```

Comment:
Is `apply` a v3 feature only?

Reply:
Nope. It was added in twig v1.40 https://twig.symfony.com/doc/1.x/tags/apply.html